packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -480,43 +483,177 @@ export class ResourceService {
     return resource;
   }
 
-  async createResourceEntitiesFromExistingResource(
-    args: CreateResourceEntitiesArgs,
+  async createModelGroupService(
```

Comment:
we should not use "modelgroup" in the server. This is a name of the concept used in the client to group models - meaning using a resource to hold entities